### PR TITLE
Add deployment to federation clientset

### PR DESCRIPTION
**Pull request by mwielgus**
Only hack/update-codegen.sh was updated manually.

cc: @quinton-hoole @nikhiljindal @kubernetes/sig-cluster-federation

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/34023)

### CODE REVIEW

File: federation/client/clientset_generated/federation_release_1_5/typed/extensions/v1beta1/generated_expansion.go [hunk 1]
```diff
@@ -16,6 +16,8 @@ limitations under the License.
 
 package v1beta1
 
+type DeploymentExpansion interface{}
```
Comment (nikhiljindal) [inline federation/client/clientset_generated/federation_release_1_5/typed/extensions/v1beta1/generated_expansion.go]: This should have a Rollback method: https://github.com/kubernetes/kubernetes/blob/a44d2f8009ab76d190ad9572e4aca661d5b6c8a3/pkg/client/clientset_generated/release_1_5/typed/extensions/v1beta1/deployment_expansion.go#L22.

Add a deployment_expansion.go file (same as the one mentioned above)

**Comment by nikhiljindal** [comment]
Thanks, looks good. Just one comment.
Need to add a deployment_expansion.go file.

**Comment by nikhiljindal** [comment]
@k8s-bot federation gce e2e test this

File: federation/client/clientset_generated/federation_release_1_5/typed/extensions/v1beta1/generated_expansion.go (hunk 1 above)
Comment (mwielgus) [inline federation/client/clientset_generated/federation_release_1_5/typed/extensions/v1beta1/generated_expansion.go]: Added.

**Comment by mwielgus** [comment]
Comment addressed. Assuming "Thanks, looks good." meant LGTM. Applying label.

**Pull request merged**